pkgs/applications/editors/bonzomatic/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, makeWrapper, fetchFromGitHub, cmake, alsaLib, mesa_glu, libXcursor, libXinerama, libXrandr, xorgserver }:
+
+let
+  name = "bonzomatic";
```

Comment:
By using a recursive attribute set as derivation argument, this is not required. You can see the `rec` keyword.